actors/miner/src/ext.rs

```diff
@@ -40,16 +42,16 @@ pub mod market {
     }
 
     #[derive(Serialize_tuple, Deserialize_tuple, Clone)]
-    pub struct VerifiedDealInfo {
+    pub struct ActivateDealInfo {
```

Comment:
Please match the original name of `ActivatedDeal`.

Reply:
Done.